### ISSUE TITLE
Create a CLI using the API

### ISSUE THREAD

**Issue by sudo-nan0-RaySK** [tushar1210/manga-node#43]
It will be nice to have a CLI to interact with this API.

**Comment by tushar1210** [comment]
Yes @sudo-nan0-RaySK it will be great! Any particular cli which you may have in mind?

**Comment by sudo-nan0-RaySK** [comment]
If you could list out what all this API is capable of doing, I might come up with something.

**Comment by tushar1210** [comment]
With the ban of kissmanga, we have decided to rebuild this project taking mangasee123.com as base. 
The Following APIs will be ready soon:
```
[GET]
- /manga/<manganame> 
- /manga/chapters 
- /manga/hot-updates
- /manga/search
- /manga/<manganame>/<chapter>
```

Also, I will update README.md as soon as service is available.
